src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/PendingBQWriter.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.gvs.ingest;
```

Comment:
Maybe put this in the common BQ package?  Is there anything specific to ingest in this class?

Reply:
Yep. Wanted to make sure it was generic first.